### ISSUE TITLE
New to python_error opening GUI

### ISSUE THREAD

**Issue by prinsrohen** [mr-sheg/orpl#3]
Hi,

I am collecting a lot of Raman data and came across your baseline correction software. Thanks for your contribution to open source coding.

Unfortunately, I am new to python.... :-)

I get this error pasted below upon launching Orpl. I spent some time trying to make the changes, but each time, there is a knock-on effect with another .py

If I make the change, it can launch the program, but then there is an issue with saving the processed spectrum. Is there something that I need to install, or have installed incorrectly to be getting this issue at the start?

Thanks for your help.

C:\WINDOWS\system32>python -m orpl
Traceback (most recent call last):
  File "C:\Users\rp2061\AppData\Roaming\Python\Python310\site-packages\orpl\gui\orplGUI.py", line 244, in selected_file_changed
    detail_str = yaml.dump(spectrum.metadata.details, Dumper=yaml.RoundTripDumper)
  File "C:\Users\rp2061\AppData\Roaming\Python\Python310\site-packages\ruamel\yaml\main.py", line 1251, in dump
    error_deprecation('dump', 'dump', arg="typ='unsafe', pure=True")
  File "C:\Users\rp2061\AppData\Roaming\Python\Python310\site-packages\ruamel\yaml\main.py", line 1039, in error_deprecation
    raise AttributeError(s, name=None)
AttributeError:
"dump()" has been removed, use

  yaml = YAML(typ='unsafe', pure=True)
  yaml.dump(...)

instead of file "C:\Users\rp2061\AppData\Roaming\Python\Python310\site-packages\orpl\gui\orplGUI.py", line 244

        detail_str = yaml.dump(spectrum.metadata.details, Dumper=yaml.RoundTripDumper)

**Comment by prinsrohen** [comment]
just to clarify, the GUI opens but when I try to select a .txt file, it gives me the error.

**Comment by DrSheg** [comment]
Hi!
Thanks a lot for pointing the issue to me, it comes from a new version of ruamel (library I use in orpl to parse YAML data). Long story short, I resolved the issue in the update v1.0.10a that I just pushed, to fix this either

```shell
pip install "ruamel.yaml<0.18.0"
```

In your current environment where you installed orpl or force an update by

```shell
pip install --upgrade orplib
```
